content/en/docs/reference/access-authn-authz/controlling-access.md

```diff
@@ -61,9 +61,18 @@ While Kubernetes uses `usernames` for access control decisions and in request lo
 it does not have a `user` object nor does it store usernames or other information about
 users in its object store.
 
+## Prioritization and fairness
+
+After the request is authenticated as coming from a specific user, the
+request is subject to prioritization and fairness if the
+RequestManagement feature is enabled.  This needs to be inserted
```

Comment:
```suggestion
RequestManagement feature is enabled.  This happens
```

Reply:
I don't think there is any preferred or recommended workflow for images; I wish there were.